core/templates/app-events/event-bus.service.spec.ts

```diff
@@ -55,11 +57,21 @@ describe('Event Bus Group', () => {
   it('should throw uncaught errors', waitForAsync(() => {
     spyOn(Subject.prototype, 'subscribe').and.callFake(
       (f) => {
+        // This throws "Argument of type '(f: PartialObserver<void> |
```

Comment:
can you explain why there is no other way to solve this?

Reply:
I have tried using these arguments defined in the class itself to identify the type of `f` but the error still persists that `f` is not callable. 
![image](https://user-images.githubusercontent.com/60519303/126980617-f814da17-cc99-4624-be87-b1bbecb13697.png)

Reply:
hmm.. alright. I think its fine since its a test